src/components/pages/Picking.tsx

```diff
@@ -0,0 +1,101 @@
+import * as ImagePicker from 'expo-image-picker';
+import React, { useEffect, useState } from 'react';
+import { Platform, StyleSheet, View } from 'react-native';
+import { Button, Image, Text } from 'react-native-elements';
+
+import { Page, TabPage } from './TabPage';
+
+const styles = StyleSheet.create({
+  lead: {
+    fontSize: 35,
+    color: '#fff',
+    marginBottom: 30,
+  },
+  select: {
+    flexDirection: 'row',
+    justifyContent: 'space-around',
+    marginBottom: 30,
+  },
+  or: {
+    fontSize: 20,
+    color: '#fff',
+    marginTop: 6,
+    marginLeft: 20,
+    marginRight: 20,
+  },
+  image: {
+    width: 200,
+    height: 200,
+    marginBottom: 30,
+  },
+  upload: {
+    backgroundColor: '#ff1493',
+  },
+});
+
+const Component: React.FC = () => {
+  const [image, setImage] = useState<string>('');
```

Comment:
好みかもしれないのですが、、、
設定されていないかどうかを空文字で判定するよりは、`undefined`で型として明示したほうがよいのかな、と思いました。意見を聞いてみたいところではあります。

```suggestion
  const [image, setImage] = useState<string | undefined>();
```

Reply:
私も最近(確か他のプラグイン実装を見た時から)undefined(null)で定義するようにしてます。

Reply:
JS書きなれてない人には空文字が分かり易そうなので変更なしで行きます。

Reply:
一般的な話としては、`undefined`が好まれてるみたいですかね。

https://typescript-jp.gitbook.io/deep-dive/recap/null-undefined

> TypeScriptチームは、nullを使いません: TypeScriptコーディングガイドライン。 そして、問題は起きていません。Douglas Crockfordはnullは良くない考えであり、誰もがundefinedだけを使うべきだと考えています。

https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#null-and-undefined